chain/events/filter/index.go

```diff
@@ -698,11 +698,12 @@ func (ei *EventIndex) GetMaxHeightInIndex(ctx context.Context) (uint64, error) {
 	return maxHeight, err
 }
 
-func (ei *EventIndex) IsHeightProcessed(ctx context.Context, height uint64) (bool, error) {
-	row := ei.stmtIsHeightProcessed.QueryRowContext(ctx, height)
-	var exists bool
-	err := row.Scan(&exists)
-	return exists, err
+func (ei *EventIndex) IsHeightPast(ctx context.Context, height uint64) (bool, error) {
```

Comment:
@rvagg 

This changed is to handle:

1) null blocks: The Event Index will never see null blocks - so the best metric for "have you already indexed this height" really is to ensure that we've indexed a height greater than it
2) holes in the event Index caused by hairy edge cases (Event Index sees a block but node crashes before it is indexed). We need auto-repair/auto-backfilling in the Index.